### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by OlgaMorales**

### CODE REVIEW

**Comment by amaurymartiny** [comment]
Thank you! I'll have a look tonight

File: App/localization/languages/es.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "about_where_does_data_come_from_title": "¿De dónde vienen los datos?",
+  "about_where_does_data_come_from_message_1": "Los datos de calidad del aire provienen de",
+  "about_where_does_data_come_from_link_1": "WAQI",
+  "about_were_does_data_come_from_message_2": "en la forma de niveles de PM2.5 AQI que generalmente se actualizan cada hora y se convierten en niveles directos de PM2.5 mediante la aplicación",
+  "about_why_is_the_station_so_far_title": "¿Por qué la estación está tan lejos de mi ubicación actual?",
+  "about_why_is_the_station_so_far_message": "Dado que las estaciones que miden y comunican los resultados de la calidad del aire cada hora son costosas, los datos aún se limitan a regiones bien desarrolladas y ciudades más grandes de todo el mundo. Si está lejos de un centro urbano más prominente, los resultados probablemente no serán tan precisos. Lo más probable es que su aire sea mejor en ese caso al menos!",
+  "about_weird_results_title": "Los resultados son extraños o inconsistentes con otras fuentes.!",
```
Comment (amaurymartiny) [inline App/localization/languages/es.json]: Is `.!` a typo?

File: App/localization/languages/es.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  "error_screen_error_cannot_load_cigarettes": "No logramos\ncargar tus\ncigarrillos.",
+  "error_screen_choose_other_location": "Elige otra ubicación",
+  "error_screen_error_description": "Existe un problema con nuestra base de datos, o no existen estaciones de monitoreo de aire cerca. Intente más tarde ",
+  "error_screen_error_message": "Error: {{errorText}}",
+  "home_station_too_far_message": "No logramos encontrar una estación cercana.\nLos resultados pueden ser inexactos a esta distancia.",
+  "home_share_title": "¿Sabía que puede estar fumando hasta 20 cigarrillos por día, solo por vivir en una gran ciudad?",
+  "home_share_message": "Maldición! 'hoy fumé'' {{cigarettes}} cigarrillos respirando el aire urbano. ¿Y tú? Encuentralo aquí: https://shootismoke.github.io",
+  "home_header_air_quality_station_distance": "Estación de calidad del aire: {{distanceToStation}}km de distancia",
+  "home_btn_why_is_station_so_far": "¿Por qué la estación está tan lejos??",
+  "home_btn_see_detailed_info": "Ver informacion detallada",
```
Comment (amaurymartiny) [inline App/localization/languages/es.json]: On smaller screens, this text goes on 2 lines. Would it be possible to find a shorter variant?

File: App/localization/languages/es.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  "home_btn_more_details": "Más detalles",
+  "home_btn_faq_about": "Faq/Acerca de",
```
Comment (amaurymartiny) [inline App/localization/languages/es.json]: This text is slightly too long, can we make it a bit shorter? We could either use Faq, or Acerca de

**Comment by OlgaMorales** [comment]
Sure! I'll make some modifications.

**Comment by OlgaMorales** [comment]
I wanted to see the changes locally, but I couldn't open the app, I followed the steps which are in the doc,
it was showed only one picture once started the project.

**Comment by lucienbl** [comment]
Hey @OlgaMorales ! Did you create a config file from the template and replaced the API field with your own key? Can you send the log?

**Comment by OlgaMorales** [comment]
hi @lucienbl where that file should be created? I just modified the the app.json file where I put my own tokens, but as I could not see my local changes then I tried it on expo  staging  release channel, could you guide me please?

**Comment by lucienbl** [comment]
Hey @OlgaMorales , why couldn't you see your local changes? :thinking: The app.json is right, you have to create this file and add your tokens there. Once you've done this just run `yarn && yarn start` which should start the development server. Then on your device download the "Expo" app from the Play Store and flash the QR code from your console :wink:

**Comment by OlgaMorales** [comment]
@lucienbl you are right I was trying another way, however if i have my phone with Spanish language as default is it not supposed that should I see everything in Spanish ?, as I see it in English yet.

**Comment by lucienbl** [comment]
@OlgaMorales wait for merging, pull again and restart app :wink:

**Comment by amaurymartiny** [comment]
@OlgaMorales Thanks for this PR! I just integrated your Spanish translations, if you `git checkout master && git pull` and re-run the app, it should show it in Spanish. Please tell us if it still doesn't work 🙏

**Pull request merged**